src/editview/Cargo.toml

```diff
@@ -34,4 +34,4 @@ serde_derive = "1"
 serde_json = "1"
 syntect = "3"
 tokio = "0.1"
-xrl = { path="/home/rasmus/Projects/xrl" }
\ No newline at end of file
+xrl = { git = "https://github.com/Cogitri/xrl.git", branch = "gxi-changes" }
```

Comment:
BTW, although not important here, I'd recommend you for future contributions to any OSS project to add a newline to end of the file, that makes for prettier diffs :)